packages/blocks/src/components/block-hub.ts

```diff
@@ -192,7 +192,7 @@ export const BLOCKHUB_FILE_ITEMS = [
 const styles = css`
   affine-block-hub {
     position: absolute;
-    z-index: 1;
+    z-index: 3;
```

Comment:
Do we have sementic css variable for this? Maybe it could be better to use the variable together with `calc`.

Reply:
We have some variables like`--affine-z-index-xx`, but I used the magic number `3` here because I found that the z-index of &lt;edgeless-toolbar&gt; was written as 3, but because it was nested too deeply, I didn’t dare to modify it easily. After that, we should change all the z-index index is unified into variables.